specification/machinelearningservices/resource-manager/Microsoft.MachineLearningServices/preview/2024-07-01-preview/workspaceRP.json

```diff
@@ -4559,6 +4654,181 @@
         }
       }
     },
+    "EndpointModelDeprecationProperties": {
+      "type": "object",
+      "properties": {
+        "fineTune": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "fineTuneOn": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "inference": {
+          "format": "date-time",
+          "type": "string"
+        },
+        "inferenceOn": {
+          "format": "date-time",
+          "type": "string"
+        }
+      }
+    },
+    "EndpointModelProperties": {
+      "type": "object",
+      "properties": {
+        "capabilities": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "deprecation": {
+          "$ref": "#/definitions/EndpointModelDeprecationProperties"
+        },
+        "finetuneCapabilities": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "format": {
+          "type": "string"
+        },
+        "isDefaultVersion": {
+          "type": "boolean"
+        },
+        "lifecycleStatus": {
+          "type": "string"
+        },
+        "maxCapacity": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "name": {
+          "type": "string"
+        },
+        "skus": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/EndpointModelSkuProperties"
+          }
+        },
+        "systemData": {
+          "$ref": "../../../../../common-types/resource-management/v3/types.json#/definitions/systemData"
+        },
+        "version": {
+          "type": "string"
+        }
+      }
+    },
+    "EndpointModelSkuCapacityProperties": {
+      "type": "object",
+      "properties": {
+        "default": {
+          "format": "int32",
+          "type": "integer"
+        },
+        "maximum": {
+          "format": "int32",
+          "type": "integer"
+        }
+      }
+    },
+    "EndpointModelSkuProperties": {
+      "type": "object",
+      "properties": {
+        "capacity": {
+          "$ref": "#/definitions/EndpointModelSkuCapacityProperties"
+        },
+        "connectionIds": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        },
+        "deprecationDate": {
```

Comment:
>deprecationDate [](http://example.com/codeflow?start=9&length=15)

What is the difference between deprecationDate and deprecationOn ? You should add descriptions to properites

Reply:
accidently include a property that should be obsolete, will remove it.